enterprise/models.py

```diff
@@ -1365,6 +1365,50 @@ def __repr__(self):
         return self.__str__()
 
 
+@python_2_unicode_compatible
+class LicensedEnterpriseCourseEnrollment(TimeStampedModel):
+    # Would the license_uuid be a primary_key on this(?)
+    license_uuid = models.UUIDField(
+        primary_key=False,
+        editable=False,
+        null=False
+    )
+
+    enterprise_course_enrollment = models.OneToOneField(
+        EnterpriseCourseEnrollment,
+        blank=False,
+        null=False,
```

Comment:
This should be `null=True` - not every `EnterpriseCourseEnrollment` will have a related `LicensedEnterpriseCourseEnrollment`.

Reply:
Oh I thought this would have done the inverse where every `LicensedEnterpriseCourseEnrollment` has n `EnterpriseCourseEnrollment`